INSTALL.md

```diff
@@ -6,9 +6,13 @@ Build Requirements
 
 To compile Coq yourself, you need:
 
-- [OCaml](https://ocaml.org/) (version >= 4.05.0)
+- [OCaml](https://ocaml.org/) (version >= 4.05.0, < 4.12.0)
   (This version of Coq has been tested up to OCaml 4.10.0)
```

Comment:
It's not been tested with 4.11?

Reply:
4.11.0 was just released. It doesn't look like we're testing it yet. @ejgallego @SkySkimmer might confirm. We should anyway "fix" this and make sure that Coq 8.12.1 is compatible with OCaml 4.11.0.

Reply:
I will wait for #12972 to land before rebasing this PR.